src/parsita/options.py

```diff
@@ -17,7 +17,7 @@ def default_handle_literal(literal: Any):
 
 def wrap_literal(literal: Input):
     from .parsers import LiteralParser
-    return LiteralParser(literal)
+    return LiteralParser((literal,))
```

Comment:
It is actually `def wrap_literal(literal: Input)` that is wrong. It should be `def wrap_literal(literal: Sequence[Input])`. A `LiteralParser` tests a sequence of `Input`s and returns a sequence of `Input`s when matching. But this has made me realize that this may be kind of annoying to use on streams that are already tokenized (where you typically deal with a single token, not a sequence of them). I have never used Parsita this way, but I suspect that is what you are doing. If the current behavior is bad for your use case, let's talk about adding an `ElementParser` or `TokenParsers` context.

Reply:
Yeah, you’re right, I aim to try to use it for a tokenized stream. Though I haven’t written any user parser code yet, just walking around seeing how to do it in a better way, so I don’t know yet if this case is annoying ot it would be okay.

So I’ll change `def wrap_literal(literal: Sequence[Input])` for now?

Reply:
Sound good to me.

I'm sure you know, but to anyone arriving on this issue via a search for `parsita tokenizer`: one of the nice things about parser combinators is that they don't need tokenizers. They can chew directly on the input stream with `lit` and `reg` acting as "on the fly" tokenizers. There are some organizational and performance benefits to having a separate tokenize phase, but it is typically easier to work without them.